extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -77,9 +77,6 @@ static public function onArticleViewHeader( &$article, &$outputDone, &$useParser
 				return true;
 			}
 
-			// article exists or existed
-			$app->wg->SuppressPageHeader = true;
-
 			$wallMessage = WallMessage::newFromTitle( $mainTitle );
 			$isDeleted = !$wallMessage->isVisible( $app->wg->User );
 			$showDeleted = ( $wallMessage->canViewDeletedMessage( $app->wg->User )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)